CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 IMPROVEMENTS:
 * Upgrade Terraform Plugin SDK to v2
 * Add support for client controlled consistency on Vault Enterprise ([#1188](https://github.com/hashicorp/terraform-provider-vault/pull/1188))
+* `resource_jwt_auth_backend_role`: Add field to disable `bound_claims` comma separation ([#1200](https://github.com/hashicorp/terraform-provider-vault/pull/1200))
```

Comment:
I wonder if we should add the name of the new field? Also I think we should use the same language  that is used in the `disable_bound_claims_parsing` rather than the bit about comma separation (comma delimiting).

Reply:
Updated to `Add field `disable_bound_claims_parsing` to disable bound claim value parsing, which is useful when values contain commas`